bms-monitor.js

```diff
@@ -1,18 +1,39 @@
-const {expect} = require('chai');
+const {limits}=require('./inputParameterLimits');
+const {outPutLogs}=require('./outputLogs');
 
-function batteryIsOk(temperature, soc, charge_rate) {
-    if (temperature < 0 || temperature > 45) {
-        console.log('Temperature is out of range!');
-        return false;
-    } else if (soc < 20 || soc > 80) {
-        console.log('State of Charge is out of range!')
-        return false;
-    } else if (charge_rate > 0.8) {
-        console.log('Charge rate is out of range!');
-        return false;
+function batteryStatus(limits,outPutLogs)
+{
+        // function to know if breach is Higher or lower than given limits
+    function checkBreach(input,parameterRange,inputParameterType){

```

Comment:
Good abstraction to convert input parameter + range into a message :+1: 
Think of a better name for this function? So I can know what the function returns by reading its name, without having to go thru all of its code

Reply:
Ok sir